src/test/java/htsjdk/samtools/SAMFileWriterFactoryTest.java

```diff
@@ -220,6 +250,28 @@ private void verifyWriterOutput(File outputFile, ReferenceSource refSource, int
         Assert.assertTrue(count == nRecs);
     }
 
+    private void verifyWriterOutput(Path output, ReferenceSource refSource, int nRecs, boolean verifySupplementalFiles) throws IOException {
+        if (verifySupplementalFiles) {
+            final Path index = SamFiles.findIndex(output);
+            final Path md5File = IOUtil.addExtension(output, ".md5");
+            Assert.assertTrue(Files.size(index) > 0);
```

Comment:
Maybe extract this out as a helper method since it's duplicated.  Although there's a ton of duplicated code in these test files anyway...